tests/smoke/jupyter/issue-12374.test.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * issue-12374.test.ts
+ * 
+ * https://github.com/quarto-dev/quarto-cli/issues/12374
+ *
+ * Copyright (C) 2023 Posit Software, PBC
+ */
+
+import { quarto } from "../../../src/quarto.ts";
+import { test } from "../../test.ts";
+import { assertEquals } from "testing/asserts";
+import { noErrors } from "../../verify.ts";
+
+test({
+  name: "jupyter:issue-12374.test.ts",
+  context: {},
+  execute: async () => {
+    // https://github.com/quarto-dev/quarto-cli/issues/12374
+    await quarto(["render", 
+      "docs/jupyter/issue-12374.ipynb",
+      "--no-execute-daemon", "--execute"]);
+  },
+  verify: [noErrors],
+  type: "smoke",
+});
```

Comment:
FWIW, we have a `testQuartoCmd()` wrapper for this, which has the benefits of having a Promise.race() for timeout in case the test hangs for some reason.

We also have another `testRender()` wrapper around it available. This wrapper has the benefit of a cleanup in the teardown step to avoid leftovers in our test runs.